### ISSUE TITLE
Problem running update-rest-api

### ISSUE THREAD

**Issue by tyagow** [localstack/localstack#3261]
# Type of request: This is a ...
[x ] bug report
[ ] feature request

# Detailed description
Having localstack pro I run update-rest-api and I get the following message:
```
╰─± aws  --endpoint=http://localhost:4566  apigateway update-rest-api --rest-api-id 7heedya7mb 

An error occurred (400) when calling the UpdateRestApi operation: 
```
## Expected behavior
run `awslocal update-rest-api ...` and it works

## Actual behavior
when running the command I get:
`An error occurred (400) when calling the UpdateRestApi operation: `
and I get the following error in the localstack log
`2020-11-20T19:31:51:INFO:localstack.services.awslambda.lambda_api: The resource you requested does not exist.`

# Steps to reproduce
`AWS_PROFILE=local DATA_DIR=/tmp/localstack/data2 PORT_WEB_UI=8081 DEBUG=1 LOCALSTACK_API_KEY=REDACTED localstack start`
`chalice deploy local`
`# Setup the base chalice app with just one function`

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)
...

┆Issue is synchronized with this [Jira Bug](https://localstack.atlassian.net/browse/LOC-338) by [Unito](https://www.unito.io/learn-more)

**Comment by tyagow** [comment]
Reference of missing API - https://docs.aws.amazon.com/cli/latest/reference/apigateway/update-rest-api.html

**Comment by whummer** [comment]
Thanks for reporting @tyagow . This should be fixed in #3288 - can you please pull the `latest` image and give it another try? Please report here if the problem persists. Thanks

**Comment by tyagow** [comment]
> Thanks for reporting @tyagow . This should be fixed in #3288 - can you please pull the `latest` image and give it another try? Please report here if the problem persists. Thanks

Thank you, I will test it today and report back with results.
